Rocket.Chat/Resources/en.lproj/Localizable.strings

```diff
@@ -116,6 +116,11 @@
 "subscriptions.channels" = "Channels";
 "subscriptions.direct_messages" = "Direct Messages";
 "subscriptions.you" = "you";
+"subscriptions.messages" = "Messages";
```

Comment:
@rafaelks `subscriptions.messages` is missing on the other languages files

Reply:
✅